File: composeApp/src/androidMain/kotlin/com/app/starleet/intent/LactateIntent.kt
package com.app.starleet.intent

sealed class LactateIntent {

    data class AddScan(
        val value: Double,
        val sweat: String,
        val change: Double
    ) : LactateIntent()
    data class SelectHistoryDate(val dateMillis: Long) : LactateIntent()
    object LoadLactate : LactateIntent()
}

File: iosApp/iosApp/Intent/LactateIntent.swift
//
//  LactateIntent.swift
//  iosApp
//
//  Created by KBS on 3/3/26.
//

import Foundation

enum LactateIntent {
    case addScan(value: Double, sweat: String, change: Double)
    case selectHistoryDate(Date)
    case loadLactate
}
